tests/testthat/test_histof.R

```diff
@@ -1,5 +1,4 @@
 context("Histocompatibility")
```

Comment:
I've deleted both these lines because they introduce a warning when running the tests.

Reply:
apagar estas linhas